### PR TITLE
Engdocs 1969

### PR DESCRIPTION
**Pull request by aevesdocker**
Addresses: https://github.com/docker/docs/issues/18809 and https://github.com/docker/docs/issues/18700

Adds version notes for major Docker Compose functionality additions going back to June 2023. Establishes a pattern of how we can do this going forward with new releases

### CODE REVIEW

**Review by glours — APPROVED** [review]
LGTM 👍

File: content/compose/compose-file/05-services.md [hunk 1]
```diff
@@ -222,8 +222,7 @@ Services can only access configs when explicitly granted by the `configs` attrib
 Compose reports an error if `config` doesn't exist on the platform or isn't defined in the
 [`configs` top-level element](08-configs.md) in the Compose file.
 
-There are two syntaxes defined for configs. To remain compliant to this specification, an implementation
-must support both syntaxes. Implementations must allow use of both short and long syntaxes within the same document.
+There are two syntaxes defined for configs.
```
Comment (dvdksn) [inline content/compose/compose-file/05-services.md:225]: ?

```suggestion
There are two syntaxes defined for configs: a short syntax and a long syntax.
```

File: content/compose/compose-file/05-services.md [hunk 2]
```diff
@@ -1154,7 +1147,7 @@ There is a performance penalty for applications that swap memory to disk often.
 
 ## network_mode
 
-`network_mode` sets a service container's network mode. Available values are platform specific, but Compose defines specific values which must be implemented as described if supported:
+`network_mode` sets a service container's network mode.
```
Comment (dvdksn) [inline content/compose/compose-file/05-services.md:1150]: Is `service:{name}` the same as a container network described here? https://docs.docker.com/network/#container-networks
Where the container gets the networking stack of another container.
Would it be useful to expand and/or add a link?

File: content/compose/compose-file/05-services.md [hunk 3]
```diff
@@ -1500,7 +1491,6 @@ section of the Docker run reference page.
 
 `runtime` specifies which runtime to use for the service’s containers.
 
-The value of `runtime` is specific to the implementation.
 For example, `runtime` can be the name of [an implementation of OCI Runtime Spec](https://github.com/opencontainers/runtime-spec/blob/master/implementations.md), such as "runc".
```
Comment (dvdksn) [inline content/compose/compose-file/05-services.md:1494]: We could call out that `runc` is the default, and to use a different runtime you need to install the shims and/or configure the daemon as described here: https://docs.docker.com/engine/alternative-runtimes/

File: content/compose/compose-file/05-services.md [hunk 4]
```diff
@@ -1118,13 +1105,53 @@ logging:
...
+
 ## network_mode
 
-`network_mode` sets a service container's network mode. Available values are platform specific, but Compose defines specific values which must be implemented as described if supported:
+`network_mode` sets a service container's network mode. 
 
 - `none`: Turns off all container networking.
 - `host`: Gives the container raw access to the host's network interface.
-- `service:{name}`: Gives the containers access to the specified service only.
+- `service:{name}`: Gives the containers access to the specified service only. For more information, see [Container networks](../../network/_index.md#container-networks).
```
Comment (dvdksn) [inline content/compose/compose-file/05-services.md:1154]: does this lgty @glours ?

**Review by dvdksn — DISMISSED** [review]
LGTM, just would like confirmation that the container networks suggestion I proposed is actually the correct interpretation

**Review by dvdksn — APPROVED** [review]
LGTM

**Pull request merged**